lib/filters.js

```diff
@@ -50,13 +50,15 @@ export const refFilter = (data, { domain, domainOnly, raw }) => {
   const links = {};
 
   const isValidRef = referrer => {
-    return (
-      referrer !== '' && referrer !== null && !referrer.startsWith('/') && !referrer.startsWith('#')
-    );
+    return referrer !== null && !referrer.startsWith('/') && !referrer.startsWith('#');
   };
 
   const cleanUrl = url => {
     try {
+      if (url === '') {
+        return 'direct';
```

Comment:
Hey, how about this?
```suggestion
        return 'Direct / None';
```
The referrers might not have been direct, It could be because of ad-blockers or other extensions which remove referrers. I feel adding `None` to specify that we do not have any information about the referrer is a better thing to do.